src/details/ArborX_MinimumSpanningTree.hpp

```diff
@@ -702,9 +702,10 @@ struct MinimumSpanningTree
       // NOTE could perform the label tree reduction as part of the update
       updateComponentsAndEdges(space, component_out_edges, labels, edges,
                                num_edges);
-      num_components =
-          static_cast<int>(n) -
-          Kokkos::create_mirror_view_and_copy(Kokkos::HostSpace{}, num_edges)();
```

Comment:
Was this a bug?

Reply:
Not really in the sense you get the right answer.  The change here are about using the execution space provided to transfer to the host instead of the default one.